### DISCUSSION TITLE
alist Aliyundrive Open 自建客户端使用Refresh Token Callback,Oauth的令牌链接是什么?

### DISCUSSION THREAD

**Discussion by Mixlining** [AlistGo/alist#6072] [Q&A]
使用了自己的开发者账号,建了应用.
在 [alist](https://alist.nn.ci/tool/aliyundrive/request.html) 上使用"use my own cilent" 获取到了refresh token(创建应用时回调网址已经填写了 https://alist.nn.ci/tool/aliyundrive/callback).
请问Oauth令牌链接是哪一个? 我使用https://api.xhofe.top/alist/ali_open/token 或者 https://api-cf.nn.ci/alist/ali_open/token 都不行

**Comment by DaPoHou** [comment]
同问。。。使用自己的开发者账号后该如何设置？

**Comment by tuimao615** [comment]
我是一直报错InvalidClientId: invalid client_id，根本得不到刷新令牌，但是手机扫码使都已经显示出我创建的应用头像和名称

**Comment by decade20092009** [comment]
同问
